README.md

```diff
@@ -2661,6 +2661,23 @@ Other Style Guides
     $sidebar.find('ul').hide();
     ```
 
+  <a name="jquery--attach-events"></a><a name="25.5"></a>
+  - [25.5](#jquery--attach-events) Avoid mess up with the events queue.
+
+    ```javascript
+    // bad
+    $('.sidebar').off('click').on('click', (e) => {
+     console.log('dont do this again!!');
+    });
+
+    // good
+    $('.sidebar').on('click', (e) => {
+     e.stopPropagation();
```

Comment:
generally speaking, stopping propagation is an impolite thing to the rest of the DOM, and should not be done lightly. Can you elaborate on this a bit?